### ISSUE TITLE
Issue with model loading (0.6.1)

### ISSUE THREAD

**Issue by valmirselmani** [bab2min/tomotopy#36]
Hi @bab2min 

Something seems wrong with the newest version 0.6.1.

models created with 0.6.1 cannot be loaded with 0.6.1, it throws an exception "Exception: 'lda.model.1000.bin' is not valid model file"

I saw that you changed the model file format: _"Since version 0.6.0, the model file format has been changed. Thus model files saved in version 0.6.0 or later are not compatible with versions prior to 0.5.2."_

**training (this works)**
```python
mdl = tp.LDAModel(tw=tp.TermWeight.IDF, k=1000)
for description in descriptions:
    ch = description.split()
    mdl.add_doc(ch)
mdl.burn_in = 100
mdl.train(0)

print('Training...', file=sys.stderr, flush=True)
for i in range(0, 1000, 10):
    mdl.train(10)
    print('Iteration: {}\tLog-likelihood: {}'.format(i, mdl.ll_per_word))

print('Saving...', file=sys.stderr, flush=True)
mdl.save("lda.model.1000.bin", True)
```

**load model and infer documents (this doesn't)**
```python
mdl = tp.LDAModel.load("lda.model.1000.bin")

docs = []
for record in records:
    docs.append(mdl.make_doc(record["description_cleaned"].split()))

infered_docs = mdl.infer(docs, together=True, parallel=3)
```

**Comment by senadjukic** [comment]
I do experience the same issue. Would be nice if this could be fixed soon @bab2min

**Comment by bab2min** [comment]
Thank you for reporting the error.
I found there was a fatal error in the `save` method of topic models.
I released a new version 0.6.2 that corrected the error, and removed old versions 0.6.0 and 0.6.1.
Please upgrade tomotopy to version 0.6.2 and try again.

**Comment by valmirselmani** [comment]
I upgraded tomotopy to version 0.6.2 and I can confirm that the issue is now resolved.

Thank you for the quick response!
